.travis.yml

```diff
@@ -15,9 +15,12 @@ jobs:
       env:
         - ADDITIONAL_DEBS="curl"
         - ROS_DISTRO="melodic"
-        - BEFORE_SETUP_TARGET_WORKSPACE="pip install coverage"
-        - DOCKER_RUN_OPTS="-e TRAVIS_COMMIT=$TRAVIS_COMMIT -e COVERAGE_RCFILE=/root/target_ws/src/ros_ci_and_coverage/.coveragerc -e COVERAGE_PROCESS_START=.coveragerc -e CODECOV_TOKEN=$CODECOV_TOKEN"
-        - AFTER_RUN_TARGET_TEST="cd /root/.ros && coverage combine && coverage xml"
+        - UPSTREAM_WORKSPACE="upstream.repos"
+        - CMAKE_ARGS="-DENABLE_COVERAGE_TESTING=ON -DCMAKE_BUILD_TYPE=Debug"
+        - DOCKER_RUN_OPTS="-e TRAVIS_COMMIT=$TRAVIS_COMMIT -e CODECOV_TOKEN=$CODECOV_TOKEN"
+        # Manually trigger combine (we aren't actually calling code_coverage target)
+        - AFTER_RUN_TARGET_TEST="cd /root/.ros && python-coverage combine && python-coverage xml"
```

Comment:
Oh.. if we're not calling the code_coverage target, what does the dependency do?

Reply:
it still does the setup for nosetests - and also does the C++ stuff (not relevant to this repo - but relevant to stuff down the road)

Reply:
Also - we can use that target to call individual packages and set thresholds for them (something I've not done here yet) in an AFTER_SCRIPT